_pages/history.md

```diff
@@ -0,0 +1,100 @@
+The first commit with the new name is from September 30,
+ 2004.(<strong>[4](https://github.com/watir/watir-classic/commit/7c7a8a)</strong>) 
+
+Bret was working at Thoughtworks at this time and was introduced
+to the work Jason Huggins was doing with leveraging JavaScript in the browser for testing using
+a program that was initially called JavaScriptTestRunner,
+and would eventually became known as 
+Selenium.(<strong>[5](https://web.archive.org/web/20140519003122/http://rubyforge.org/pipermail/wtr-general/2004-August/000228.html)</strong>) 
+Fun fact: Jason made a list of potential names for his project, but when Thoughtworks needed
+  to present an agenda that included discussing this technology at a conference in Australia, Jason wasn't
+  available to provide the official name. Paul Hammant and Bret looked at the list, 
+  and Bret's background in the QA community caused him to recognize the humor in calling the project
+  Selenium. The primary expensive commercial UI testing solution at the time was owned by 
+  [Mercury Interactive](https://en.wikipedia.org/wiki/Mercury_Interactive) 
+  and Selenium is known to be a [cure for Mercury poisoning](https://www.ncbi.nlm.nih.gov/pubmed/1304229).
+  Jason returned to find his project named for him, but it's certainly proved to be an excellent brand name.
+
+Original versions of Selenium were focused on being a test framework with its own HTML table based 
+Domain Specific Language (DSL). It had some technological limitations and issues with flexibility, 
+as well as offering multiple ways of doing similar things. Its big advantage was its support
+for all of the various browsers that were starting to steal market share from Microsoft. 
+To keep up with Selenium, various groups created browser specific projects for Watir including
+[FireWatir](https://github.com/facebookarchive/watir), 
+[OperaWatir](https://github.com/operasoftware/operawatir), 
+[SafariWatir](https://github.com/redsquirrel/safariwatir), and
+[ChromeWatir](https://github.com/jijeshmohan/chromewatir)).
+
+Selenium also had the advantage of easily supporting development in multiple programming languages. 
+As a result, Watir was also adapted into Java with [Watij](https://sourceforge.net/projects/watij/) and
+.NET with [Watin](https://www.codeproject.com/Tips/658947/Watin-An-Automation-Testing-in-NET).
+
+When WebDriver was created by Simon Stewart in 2007 (also at Thoughtworks at that time), it provided
+ an alternate solution to the issues Selenium faced. It boasted a cleaner
+API and did not rely extensively on JavaScript. It was more of a toolkit 
+than a framework, and focused more on direct browser automation commands 
+than various testing features.(<strong>[6](http://www.seleniumhq.org/about/history.jsp)</strong>) 
+Selenium completely overhauled its interface for its version 2.0 release in order to adopt it. 
```

Comment:
Selenium 2.0 was the point where Selenium offered the WebDriver APIs as well as the original RC APIs. The interface wasn't so much "overhauled" as "grew an alternate set of APIs". We only just managed to get back down to one set of APIs with the 3.0 release. :)